com.atlauncher.ATLauncher.metainfo.xml

```diff
@@ -0,0 +1,103 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
This file should be submitted upstream.

Reply:
What do you mean submitted upstream?

Reply:
The reason it is not is because the run command is different.

Reply:
The launchable is a .desktop file so it doesn't matter here.